packages/main/src/MessageStrip.js

```diff
@@ -91,6 +72,23 @@ const metadata = {
 		"default": {
 			type: Node,
 		},
+
+		/**
+		 * Defines the icon to be displayed as graphical element within the <code>ui5-messagestrip</code>.
```

Comment:
Defines the content(as the user can pass not only icon)

Reply:
I'll change it